go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/jinzhu/gorm v1.9.12
 	github.com/kelseyhightower/envconfig v1.4.0 // indirect
 	github.com/lib/pq v1.3.0
-	github.com/lyft/flyteidl v0.18.10
+	github.com/lyft/flyteidl v0.18.11-0.20201124224632-b7d8ea8378ab
```

Comment:
Remove commit and point to v0.18.11 after flyteidl PR merged and released. @honnix